models/MITgcm_ocean/expand_netcdf.f90

```diff
@@ -0,0 +1,149 @@
+! Uncompress a netcdf fil
+program expand_netcdf
+
+use netcdf_utilities_mod, only: nc_open_file_readonly, nc_get_dimension_size, &
+                                nc_define_dimension, nc_create_file, &
+                                nc_get_variable, nc_close_file, nc_put_variable, &
+                                nc_define_real_variable, nc_end_define_mode, &
+                                nc_add_attribute_to_variable
+
+use types_mod,        only : r4, MISSING_R4
+
+use utilities_mod,    only : initialize_utilities, finalize_utilities
+                             
+use netcdf
+
+implicit none
+
+integer :: ncid, ncid_comp, dimid(1), dimlen, ret
+integer :: Nx,Ny,Nz
+integer :: nvars ! total number of variables in compressed file
+integer :: id, n, c! loop variables
+integer :: i,j,k, ncomp3d, ncomp2d
+character(len=NF90_MAX_NAME) :: varname
+real(r4), allocatable :: vals3d(:,:,:), vals2d(:,:), vals_comp(:)
+integer, allocatable :: Xcomp_ind(:), Ycomp_ind(:), Zcomp_ind(:)
+
+call initialize_utilities('expand_netcdf')
+
+ncid_comp = nc_open_file_readonly('compressed.nc')
+ncid = nc_create_file('expanded.nc')
```

Comment:
@hkershaw-brown this is the call that needs 64bit offset.
We're removing this expand_netcdf.f90 since it does not have the different compressions for the regular and staggered grids. This means we can punt to another pull request updating netcdf_utilities_mod to use 64 bit offset.